src/test/kotlin/nexters/admin/domain/user/administrator/AdministratorTest.kt

```diff
@@ -0,0 +1,29 @@
+package nexters.admin.domain.user.administrator
+
+import io.kotest.matchers.comparables.shouldBeGreaterThan
+import io.kotest.matchers.shouldBe
+import nexters.admin.domain.user.Password
+import nexters.admin.createNewAdmin
+import org.junit.jupiter.api.Test
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+
+class AdministratorTest {
+
+    @Test
+    fun `비밀번호 일치 여부 반환`() {
+        val password = "abcd1234"
+        val admin = createNewAdmin(password = password)
+
+        admin.isSamePassword(Password(password)) shouldBe true
+    }
+
+    @Test
+    fun `현재 시각으로 마지막 접속 시점 수정`() {
+        val prevAccessTime = LocalDateTime.now().minus(3, ChronoUnit.SECONDS)
+        val admin = createNewAdmin(lastAccessTime = prevAccessTime)
+
+        admin.updateLastAccessTime()
+        admin.lastAccessTime!! shouldBeGreaterThan prevAccessTime
```

Comment:
개인적으로 `!!`를 최대한 지양하자는 주의여서... (nullable 취지를 아예 무시하는 연산자라고 생각해요) 혹시 이건 어떤가요?
```suggestion
        admin.lastAccessTime?.shouldBeGreaterThan prevAccessTime
```
다만 lastAccessTime의 중요도가 낮다는 점, 테스트 코드에서 작성된 점이라는 걸 감안하면 꼭 수정은 안해도 될 거 같아요

Reply:
컬럼 타입 자체를 not nullable로 수정하는 방식으로 수정했습니다!